docs/event-bus/10-03-troubleshooting-knative-eventing-upgrade.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Troubleshooting Knative Eventing upgrade
+type: Troubleshooting
+---
+
+During the upgrade process, you can occasionally come across situations where the `kyma-installer` gets stuck at upgrading `knative-eventing`. This can happen because the upgrade process was unable to clear existing Knative Subscriptions, created internally by Kyma's `event-bus-subscription-controller`. 
```

Comment:
```suggestion
During the upgrade process, you can occasionally come across situations where the `kyma-installer` gets stuck at upgrading the `knative-eventing` component. This can happen because the upgrade process is unable to clear existing Knative Subscriptions created internally by Kyma's `event-bus-subscription-controller`. 
```